doc/release_notes_1.0.rst

```diff
@@ -0,0 +1,478 @@
+between Guest OS instances.
+
+USB (xDCI)
+==========
+ACRN hypervisor supports virtualized assignment of all USB xHCI and/or xDCI 
+controllers to a Guest OS from the platform.
+
+USB Mediator (xHCI and DRD)
+===========================
+ACRN hypervisor supports a virtualized USB Mediator.
+
+
+Intel® Converged Security and Management Engine  (CSME)
+=======================================================
+ACRN hypervisor supports a CSME to a single Linux, Android, or RTOS guest or 
+the Service OS even when in a virtualized environment.
+
+WiFi
+====
+ACRN hypervisor supports the passthrough assignment of the WiFi subsystem to 
+the IVI, enables control of the WiFi as an in-vehicle hotspot for 3rd party 
+devices, provides 3rd party device applications access to the vehicle, and 
+provides access of 3rd party devices to the TCU provided connectivity.
+
+IPU (MIPI CSI-2, HDMI-in)
+=========================
+ACRN hypervisor supports passthrough IPU assignment to Service OS or Guest 
+OS, without sharing.
+
+Bluetooth
+=========
+ACRN hypervisor supports bluetooth controller passthrough to a single Guest 
+OS (IVI).
+
+GVT-g for ACRN
+==============
+GVT-g for ACRN (a.k.a AcrnGT) is a feature to enable GPU sharing Service OS 
+and User OS, so both can run GPU workload simultaneously. Direct display is 
+supported by AcrnGT, where the Service OS and User OS are each assigned to 
+a different display. The display ports support eDP and HDMI. See GVT-g 
+high-level design documentation for more information.
+
+GPU - Preemption
+================
+GPU Preemption is one typical automotive use case which requires the system 
+to preempt GPU resources occupied by lower priority workloads. This is done 
+to ensure performance of the most critical workload can be achieved. Three 
+different schedulers for the GPU are involved: i915 UOS scheduler, Mediator 
+GVT scheduler, and i915 SOS scheduler.
+
+GPU - display surface sharing via Hyper DMA
+===========================================
+Surface sharing is one typical automotive use case which requires that the 
+SOS accesses an individual surface or a set of surfaces from the UOS without 
+having to access the entire frame buffer of the UOS. Hyper DMA Buffer sharing 
+extends the Linux DMA buffer sharing mechanism where one driver is able to 
+share its pages with another driver within one domain.
+
+Virtio standard is supported
+============================
+Virtio is a virtualization standard for network and disk device drivers where 
+the guest’s device driver “knows” it is running in a virtual environment, and 
+cooperates with the hypervisor. The SOS and UOS can share physical LAN network 
+and physical eMMC storage device. (See Virtio devices high-level design for 
+more information.)
+
+Device pass-through support
+===========================
+Device pass-through to UOS supported.
+OVMF support initial patches merged in ACRN: To support booting Windows as a 
```

Comment:
This should be promoted as a new section (it's under the pass-through description now).